### DISCUSSION TITLE
Get modified document when using JsonPatchOperation

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#13205] [Q&A] [answered]
Do I need to load the document after the `JsonPatchOperation` to get the modified document? This one is available `JsonPatchResult.ModifiedDocument` but it can not be serialized.

**Comment by ayende** [comment, marked as answer]
Are you referring to the new JSON Patch operation? Or patching in general?

We aren't sending the modified document back, given that you used patch, you likely want to save traffic, so we aren't sending back anything but confirmation

**Reply by marcuslindblom** [reply]
I'm referring to the new JSON Patch operation

**Reply by ayende** [reply]
You have to load the document in a separate operation

**Reply by marcuslindblom** [reply]
Ok, thanks
